Service/Runner.php

```diff
@@ -94,7 +94,7 @@ class Runner
      */
     public function __construct(
         ObjectEncryptorFactory $encryptorFactory,
-        StorageApiService $storageApi,
+        OriginalStorageApiService $storageApi,
```

Comment:
ja vim, ze jsi s tim nejak zapasil, ale me to funguje uplne normalne s Keboola\DockerBundle\Service\StorageApiService

Reply:
tak to jdu zkusit ještě jednou, zkoušel jsi pustit job z commandlajny? failovalo mi to až tam, testy tuším procházely

Reply:
tak teď to už prošlo, hm :-/